<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>本文例子</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="drawSimpleTextButton">
     <property name="text">
      <string>简单文本</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="wrapTextButton">
     <property name="text">
      <string>自动换行</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="centralTextButton">
     <property name="text">
      <string>文本居中</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="autoResizeTextButton">
     <property name="text">
      <string>自动计算大小</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="fontMetricsButton">
     <property name="text">
      <string>Font Metrics</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
